Sources/Overture/KeyPath.swift

```diff
@@ -116,13 +116,25 @@ public func mprop<Root, Value>(
 ///   - keyPath: A reference-writable key path.
 ///   - update: An update function for a given value.
 /// - Returns: A reference-mutable setter function.
+//public func mver<Root, Value>(
+//  _ keyPath: ReferenceWritableKeyPath<Root, Value>,
+//  _ update: @escaping (Value) -> Void
+//  )
+//  -> (Root) -> Void {
+//
+//    return mprop(keyPath)(update)
+//}
+
 public func mver<Root, Value>(
-  _ keyPath: ReferenceWritableKeyPath<Root, Value>,
+  _ keyPath: KeyPath<Root, Value>,
```

Comment:
what's this about?

Reply:
oops. was troubleshooting some ambiguity. we need to put these APIs in unison through the paces a bit more.